lib/rails/generators/active_interactor.rb

```diff
@@ -8,10 +8,26 @@ module Generators
     module Generator
       private
 
-      def interactor_dir
-        @interactor_dir ||= ActiveInteractor.config.rails.directory
+      def interactor_directory
+        @interactor_directory ||= ActiveInteractor.config.rails.directory
       end
     end
+
+    module GeneratesContext
+      def self.included(base)
+        base.class_eval do
+          argument :context_attributes, type: :array, default: [], banner: 'attribute attribute'
+          class_option :skip_context, type: :boolean, desc: "Whether or not to generate a context class"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Prefer single-quoted strings when you don't need string interpolation or special symbols.](https://app.codacy.com/manual/aaronmallen/activeinteractor/pullRequest?prid=4773174)